package.json

```diff
@@ -28,6 +28,7 @@
     "@mui/icons-material": "5.2.5",
     "@mui/material": "5.2.5",
     "@nervosnetwork/ckb-sdk-utils": "0.102.3",
+    "@tanstack/react-query": "4.2.3",
```

Comment:
Because if we don't add this, wagmi gives error: https://github.com/wagmi-dev/wagmi/issues/863, and adding `@tanstack/react-query` is the workaround mentioned in this issue. 
Not sure what's the cause of this and I cannot reproduce this error in codesandbox, so I followed this workaround. 

I think maybe it was caused by our version of react-query (3.39) is lower than wagmi's dependency version, but upgrading our react-query to v4 may introduce some breaking changes and may introduce extra efforts, so I did not try it for now.

Reply:
`"@tanstack/react-query"` is a dep of `wagmi` so it should be installed along with `wagmi` and works as a independent package from `react-query`

Try using `npm@8`, I guess it might be an issue from auto-install module of npm